### ISSUE TITLE
In the Azure DevOps Retrospectives page, the group name of a feedback item is announced after focusing on the 'Vote up' and 'Vote down' buttons, instead of being announced initially after navigating onto the feedback item group.

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1199]
#A11y #Accessibility issue

**Describe the bug**
In the Azure DevOps Retrospectives page, the group name of a feedback item is announced after focusing on the 'Vote up' and 'Vote down' buttons, instead of being announced initially after navigating onto the feedback item group. Screen reader users are not able to identify the name of the feedback item at the beginning of the group and it is being announced upon navigating onto each 'Vote up' and 'Vote down' buttons making the experience illogical and verbose.

**Are you using the Retrospectives Tool on-prem?**
No

**To Reproduce**

1. open the URL https://dev.azure.com/eDADPlayground/SCRUMProject .
2. Expand left navigation and select boards-->Retrospectives.
3. In Board > Vote tab, navigate to a feedback item and observe the announcement.
4. Navigate onto the 'Vote up' and the 'Vote down' buttons inside the feedback item.
5. Observe the screen reader announcement while navigating through the feedback item.

**Expected behavior**
The title of the feedback item should be announced initially while navigating onto the group, and it should not repeat the name of the group while navigating through the 'Vote up' and 'Vote down' buttons in the group since the screen reader user would understand that the buttons are present within the group.

**Screenshots**

https://github.com/user-attachments/assets/f47cc16e-dcb5-41e9-8db0-28b9b762dcfd

![Image](https://github.com/user-attachments/assets/9795dbc6-e539-470e-b18b-839ad58f250b)

**What screen type are you seeing this on ?**
Desktop 
Windows 11 - Version 24H2 (OS Build 26200.5600)
Microsoft Edge - Version 137.0.3296.16
JAWS - Version 2025.2504.89, NVDA - Version 2024.4.2.35031

**Additional context**
